mod/twitter_api/views.php

```diff
@@ -2,8 +2,6 @@
 
 return [
 	'default' => [
-		'twitter_api/sign-in-with-twitter-d.png' => __DIR__ . '/graphics/sign-in-with-twitter-d.png',
-		'twitter_api/sign-in-with-twitter-l.png' => __DIR__ . '/graphics/sign-in-with-twitter-l.png',
-		'twitter_api/sign-in-with-twitter.gif' => __DIR__ . '/graphics/sign-in-with-twitter.gif',
+		'twitter_api/' => __DIR__ . '/graphics',
```

Comment:
Wow, at this rate mapping directories seems like it will be more common than mapping files...?